app/views/admin/suppliers/lead_provider_users/delete.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Confirm deletion of <%= @lead_provider_user.full_name %></h1>
```

Comment:
Get a design review with Al and Shani on this

Reply:
Checked and done